Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -7516,11 +7524,13 @@
 											"listen": "test",
 											"script": {
 												"exec": [
-													"pm.test(\"Status is 201\", () => {\r",
-													"  pm.expect(pm.response.code).equals(201);\r",
-													"});\r",
+													"pm.test(\"Response is  201 or 404\", () => {\r",
```

Comment:
This is for another ticket, but providing blanket allowance for `404` errors could provide false positives in the test results by hiding a `404` response resulting from an actual error in the API (like a bad `id` on a GetById style request). When we address these tests in another ticket, we should probably probe the API and define Postman environment variables that state explicitly the appropriate response statuses for contact or parent-related requests (e.g. `pm.environment.set("expectedContactCreateStatus", 201)` or `pm.environment.set("expectedContactCreateStatus", 404);`).

So the action item for this comment is make sure to add this content to the ticket being created to ensure proper Postman test coverage of Contacts/Parents.

Reply:
https://tracker.ed-fi.org/browse/ODS-5921